pkg/scrape/target.go

```diff
@@ -388,7 +388,11 @@ func targetsFromGroup(tg *targetgroup.Group, cfg *config.ScrapeConfig, targets [
 				}
 
 				if pcfg, found := cfg.ProfilingConfig.PprofConfig[profType]; found && pcfg.Delta {
-					params.Add("seconds", strconv.Itoa(int(time.Duration(cfg.ScrapeInterval)/time.Second)))
+					seconds := int(time.Duration(cfg.ScrapeInterval) / time.Second)
```

Comment:
do you mind just leaving a few code comments here, I had to read it twice to understand it right, just mention that we default to the scrape interval but if the seconds are explicitly set, we use that

Reply:
no worries – i just included its patch in my rebased branch.